ecl/hql/hqlrepository.cpp

```diff
@@ -584,7 +584,7 @@ void EclRepositoryManager::processArchive(IPropertyTree * archiveTree)
     addRepository(archiveCollection, nullptr, true);
 }
 
-IEclPackage * EclRepositoryManager::resolveDependentRepository(IIdAtom * name, const char * defaultUrl, bool requireSHA)
+IEclPackage * EclRepositoryManager::queryDependentRepository(IIdAtom * name, const char * defaultUrl, bool requireSHA)
```

Comment:
I'm not sure that I understand why this function has been changed to a query* function.  I though query* functions should be returning anything that needs to be freed by the caller.  Please can you clarify.

Reply:
no, query() functions do not need freeing by the caller.  get() functions do need to be freeing.